recipes/buffer-atob-btoa/src/workflow.ts

```diff
@@ -0,0 +1,79 @@
+import type { Edit, Kinds, Range, SgNode, SgRoot } from '@codemod.com/jssg-types/main';
+import type Js from '@codemod.com/jssg-types/langs/javascript';
+import { resolveBindingPath } from '@nodejs/codemod-utils/ast-grep/resolve-binding-path';
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { removeLines } from '@nodejs/codemod-utils/ast-grep/remove-lines';
+import { removeBinding } from '@nodejs/codemod-utils/ast-grep/remove-binding';
+
+export default function transform(root: SgRoot<Js>): string | null {
+	const rootNode = root.root();
+	const bindingStatementFnTuples: [string, SgNode<Js, Kinds<Js>>, (arg: string) => string][] = [];
```

Comment:
```suggestion
	const bindingStatementFnTuples: [string, SgNode<Js>, (arg: string) => string][] = [];
```

this type can be simplified